### ISSUE TITLE
Publish 0.1.0

### ISSUE THREAD

**Issue by davglass** [ncb000gt/node-bbcode#2]
The `package.json` in the repo says version: `0.1.0`

But the latest on npm is: `0.0.2`

```
npm show bbcode version
npm info it worked if it ends with ok
npm info using npm@1.1.62
npm info using node@v0.8.11
0.0.2

npm info ok 
```

Can you publish the latest?

**Comment by ncb000gt** [comment]
Thanks for the reminder. Was working on something else and forgot to do so. I've published v0.1.1.

**Comment by davglass** [comment]
Thanks!
